xbmc/cores/AudioEngine/Utils/AEUtil.cpp

```diff
@@ -554,12 +554,7 @@ AVSampleFormat CAEUtil::GetAVSampleFormat(AEDataFormat format)
     case AEDataFormat::AE_FMT_RAW:
       return AV_SAMPLE_FMT_U8;
     default:
-    {
-      if (AE_IS_PLANAR(format))
```

Comment:
`AE_IS_PLANAR = AE_FMT_U8P|AE_FMT_S16NEP|AE_FMT_S32NEP|AE_FMT_S24NE4P|AE_FMT_S24NE4MSBP|AE_FMT_S24NE3P|AE_FMT_DOUBLEP|AE_FMT_FLOATP` and all those are already handled explicitly above.